var/spack/repos/builtin/packages/rocm-smi-lib/disable_pdf_generation_with_doxygen_and_latex.patch

```diff
@@ -0,0 +1,69 @@
+diff --git a/rocm_smi/CMakeLists.txt b/rocm_smi/CMakeLists.txt
+index 90cad13..c398b3c 100755
+--- a/rocm_smi/CMakeLists.txt
++++ b/rocm_smi/CMakeLists.txt
+@@ -140,34 +140,34 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/bindings_link
+ 
+ 
+ # Generate Doxygen documentation
+-find_package(Doxygen)
```

Comment:
Can you keep the patch minimal and just delete the lines?

I'll file yet another issue upstream that they should only do this when a variable is set...

Reply:
Oh, you have already done that :D thanks!